R/pkg/R/mllib.R

```diff
@@ -126,3 +126,27 @@ setMethod("summary", signature(object = "PipelineModel"),
               return(list(coefficients = coefficients))
             }
           })
+
+#' Fit a k-means model
+#'
+#' Fit a k-means model, similarly to R's kmeans().
+#'
+#' @param x DataFrame for training
+#' @param centers Number of centers
+#' @param iter.max Maximum iteration number
+#' @param algorithm Algorithm choosen to fit the model
+#' @return A fitted k-means model
+#' @rdname kmeans
+#' @export
+#' @examples
+#'\dontrun{
+#' model <- kmeans(x, centers = 2, algorithm="random")
+#'}
+setMethod("kmeans", signature(x = "DataFrame"),
```

Comment:
shall this match the signature of stats::kmeans in R?
https://stat.ethz.ch/R-manual/R-devel/library/stats/html/kmeans.html